cmake/CMakeLists.txt

```diff
@@ -785,6 +785,16 @@ if(NOT TARGET re2::re2)
   set(RE2_INCLUDE_DIR ${REPO_ROOT}/cmake/external/re2)
 endif()
 
+# Adding pytorch CPU info library
+# TODO  do we have to add target_include_directories to each project that uses this?
```

Comment:
perhaps not, is it enough to just call target_link_libraries(x cpuinfo)?
https://github.com/pytorch/cpuinfo/blob/5916273f79a21551890fd3d56fc5375a78d1598d/CMakeLists.txt#L213

Reply:
compilation fails without this line, don't know why. Maybe if they change PUBLIC to INTERFACE it would work